hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/ERCContractFeature.java

```diff
@@ -411,6 +419,74 @@ public void approveCryptoAllowanceAllSerials(String accountName) {
         assertNotNull(networkTransactionResponse.getReceipt());
     }
 
+    @RetryAsserts
+    @Then("I call the erc contract via the mirror node REST API for token isApprovedForAll with response true with alias accounts")
+    public void isApprovedForAllWithAliasSecondContractCall() {
+        ECDSAAccount = accountClient.createNewAccount(0, BOB);
```

Comment:
Sorry, didn't realize we should actually call `getAccount` so the account is created and cached. Calling create explicitly will create multiple accounts.

```suggestion
        ECDSAAccount = accountClient.getAccount(BOB);
```